tests/acceptance/package.json

```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "@currents/playwright": "^1.5.11",
     "@playwright/test": "1.49.1",
-    "@shopware-ag/acceptance-test-suite": "^10.2.3",
+    "@shopware-ag/acceptance-test-suite": "11.0.0",
```

Comment:
in future - dependency bump should be a separate PR

related PR which is also doing the bump #5931 linking so you're aware of each other :)